### ISSUE TITLE
python interpreter and nodeConfigureArgs not recognized

### ISSUE THREAD

**Issue by brbsix** [nexe/nexe#253]
I've been trying to use nexe on an arm64 platform (Android via [Termux](https://termux.com/)) where Python 3.5.2 is the default `python` interpreter and have had all sorts of issues. I can compile Node.js alone just fine, so I know the problem is with nexe. What follows is what I tried and the problems I encountered.

First, I configure nexe to use a different Python interpreter via `--python` or _package.json_. I get the same error described in #237. Something seems to still be using the system's default Python 3.5.2 interpreter.

Clearly telling nexe to use a different Python interpreter isn't working, so I use a symlink to set Python 2.7.12 as the system's `python` (and leave it that way). I try running nexe again, but this time none of the nodeConfigureArgs that are set in _package.json_ are passed to `./configure` and the compilation fails as a result.

However now if I pass in anything via `--python` or _package.json_ (e.g. `python2`), all of a sudden nodeConfigureArgs are passed to `./configure`... But it eventually fails with the following error:

```
----> patching Makefile
/data/data/com.termux/files/home/Development/tmp/zippyshare-node.js/node_modules/nexe/lib/exe.js:534
                pv = undefined;
                   ^

ReferenceError: pv is not defined
```

I'm not really sure what is going on here. I've been doing all this from my phone so I haven't had a chance to give it a proper examination. Seems like some sort of logic in nexe is wrong. I don't know what the proper fix is, but I was able to dig around enough to get it working for myself. Assuming Python 2 is the default `python` these are the changes that need to be made to _exe.js_:

```
501c501
<           if (isPy !== "python") {

---
>           if (isPy == "python") {
534c534
<                 pv = undefined;

---
>                 py = undefined;
```

**Comment by jaredallard** [comment]
yeah, python support is a massive hack. Monkey patches node source and is easily broken. Odd pv issues though

**Comment by calebboyd** [comment]
fixed in #308
